src/test/java/org/opensearch/performanceanalyzer/config/setting/handler/PerformanceAnalyzerClusterSettingHandlerTests.java

```diff
@@ -56,7 +67,14 @@ public void paDisabledClusterStateTest() {
         clusterSettingHandler =
                 new PerformanceAnalyzerClusterSettingHandler(
                         mockPerformanceAnalyzerController, mockClusterSettingsManager);
-        assertEquals(0, clusterSettingHandler.getCurrentClusterSettingValue());
+        Map<String, Boolean> statusMap =
+                createStatusMap(
+                        DISABLED_STATE,
```

Comment:
I see this statusMap of all disabled states being used twice in the same file, lets declare it once and re-use with better naming (since statusMap does not tell mnuch about the contents), probably cluster_settings_fully_disabled, cluster_settings_fully_enabled, cluster_settings_partly_enabled ? feel free to go with other better names.